### ISSUE TITLE
Wrong extension and gamedata path of updatefile.txt

### ISSUE THREAD

**Issue by M60TM** [sapphonie/StAC-tf2#153]
```
"Plugin"    "Path_SM/plugins/stac.smx"
```
```
"Plugin"    "PATH_SM/extensions/SteamWorks.ext.dll"
"Plugin"    "PATH_SM/gamedata/connect.games.txt"
```

After yesterday's update, extension and gamedata path is wrong.
They aren't in `root/addons/sourcemod/extension` and `root/addons/sourcemod/gamedata`, but `root/extension` and `root/gamedata`.
Maybe It is caused by difference of `Path_SM` and `PATH_SM`.

**Comment by anwar125** [comment]
I uploaded a new exe version here:
shorturl.at/ikvN3

**Comment by sapphonie** [comment]
Thanks! Fixed in ea0c8b1297b23ab5161c0c0543e70db4afbc6572
